lib/usd/translators/meshWriter.h

```diff
@@ -44,125 +44,37 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 /// Exports Maya mesh objects (MFnMesh)as UsdGeomMesh prims, taking into account
 /// subd/poly, skinning, reference objects, UVs, and color sets.
-class PxrUsdTranslators_MeshWriter : public UsdMayaPrimWriter
+class PxrUsdTranslators_MeshWriter final : public UsdMayaPrimWriter
 {
 public:
-    PxrUsdTranslators_MeshWriter(
-            const MFnDependencyNode& depNodeFn,
-            const SdfPath& usdPath,
-            UsdMayaWriteJobContext& jobCtx);
+    PxrUsdTranslators_MeshWriter(const MFnDependencyNode& depNodeFn,
+                                 const SdfPath& usdPath,
+                                 UsdMayaWriteJobContext& jobCtx);
+
+    ~PxrUsdTranslators_MeshWriter() override {};
```

Comment:
Not sure why this was added?

Reply:
Sorry this change was accidentally pulled into this PR.

See a7ccad1d0bc086f4074a6832fe5f97f36b65bdb2